cmd/shutdown.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+)
+
+const (
+	shutdownTimeout = 10 * time.Second
+)
+
+//GracefulShutdown implements releasing all resouces it got from system, finish all request handling and return responses when service stopping.
+func gracefulShutdown(done chan<- bool) error {
```

Comment:
Function returns error by signature but it doesn't return error anywhere

Reply:
fixed